client-data/js/board.js

```diff
@@ -123,6 +123,14 @@ Tools.HTML = {
 				Tools.i18n.t("keyboard shortcut") + ": " +
 				toolShortcut + ")" +
 				(Tools.list[toolName].secondary ? " [" + Tools.i18n.t("click_to_toggle") + "]" : "");
+			if(Tools.list[toolName].secondary) {
+				elem.classList.add('hasSecondary');
+				var secondaryIcon = toolIconElem.cloneNode();
+				secondaryIcon.classList.add("secondaryIcon");
+				secondaryIcon.src = Tools.list[toolName].secondary.icon;
+				elem.appendChild(secondaryIcon);
+				toolIconElem.classList.add("primaryIcon");
+			}
```

Comment:
We should probably avoid to dynamically create dom nodes here ? Can we create the img tag directly  in `board.html` and just set its src here ?

Reply:
Yep, fair point. Made that change now